src/tests/test_tls_handshake_transitions.cpp

```diff
@@ -0,0 +1,91 @@
+/*
+* (C) 2022 Jack Lloyd
+* (C) 2022 Hannes Rantzsch, René Meusel - neXenio
+*
+* Botan is released under the Simplified BSD License (see license.txt)
+*/
+
+#include "tests.h"
+
+#if defined(BOTAN_HAS_TLS)
+
+#include <botan/internal/tls_handshake_transitions.h>
+
+using namespace Botan::TLS;
+using namespace Botan_Tests;
```

Comment:
Instead of being in global namespace with a `using` this should be within `Botan_Tests` namespace